docs/kusama-social-recovery.md

```diff
@@ -0,0 +1,143 @@
+need to be aware of when using it.
+
+## Create a Recoverable Account
+
+You will use your existing account to call `createRecovery` to select a list of accounts that you
+trust to help you recover the account when you lost the private key. To create a recoverable
+account, you will be required to set a `threshold` that is the number of your friends who need to
+approve the recovery process in order to recover your account.
+
+![](/img/recovery/social-recovery-diag-1.png)
+
+First, go to [PolkadotJS->Accounts](https://polkadot.js.org/apps/#/accounts) page that shows all
+available accounts on your browser's local storage. To create a recoverable account, making sure
+that you have some KSMs to pay for the transaction fees and some will be used for reserved as well.
+
+![](/img/recovery/social-recovery-1.png)
+
+Then click the menu that besides the `send` button, and choose `Make recoverable`.
+
+![](/img/recovery/social-recovery-2.png)
+
+Now you need to provide the following information:
+
+`trusted social recovery helpers` - A list of accounts that you trust can help you when you lost the
+private key. Since setting up a recoverable account is required to lock up KSMs, better to ensure
+sure your account has enough transferrable balance for this. As you select more recovery helpers,
+more KSMs will be required to lock up.
+
+`recovery threshold` - The number of friends is required to submit a `vouchRecovery` transaction in
+order to recover the account.
+
+> Note: 1 is the minimum, but it is not recommended to set a small number. If you set 1, that means
+> any of your recovery helpers would be able to recover your account.
+
+`recovery block delay` -Once the threshold is reached, you will need to wait until the block delay
+is passed and then you can claim the recovery. This is a protection mechanism to allow the account
+owner to have enough time to check and react in case someone pretends you to start the recovery
+process.
+
+> Note: set the block delay a little longer would be better since even they have acquired enough
+> signatures to recover your account, they would still have to wait until the block delay is passed
+> in order to control your account.
+
+![](/img/recovery/social-recovery-3.png)
+
+## Recover your Account
+
+This section would be showing you how to initiate a recovery process and get back the balance that
+held in the lost account to the new account.
+
+![](/img/recovery/social-recovery-diag-2.png)
+
+The above diagram shows the whole process of recovering an account.
+
+> Note: ensuring that your new account has enough KSM for paying the transaction fees and some are
+> used for reserved when doing the recovery.
+
+First, use your new account that shows in the [Account](https://polkadot.js.org/apps/#/accounts)
+page to initiate a recovery process by clicking the "Initiate recovery for another" option that is
+in the menu besides the send button.
```

Comment:
"Navigate to the menu that is located besides the send button in the row of your new account and click the "Initiate recover for another" option.